### PR TITLE
[1.3.x] Bump kommander to 0.15.12

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

bug

**What this PR does/ why we need it**:

Bump kubecost for pvc bug fix, this is for federated kubecost (vs kommander centralized kubecost which has been bumped).
This backports to 1.3.x

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-75770
https://jira.d2iq.com/browse/COPS-6937

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE (already mentioned)
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by branden — APPROVED** [review]

File: addons/kommander/1.3/kommander.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.3-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.3-4"
```
Comment (joejulian) [inline addons/kommander/1.3/kommander.yaml:12]: Increment the addon patch version to account for the patch version change of the chart. Only increment release version when the only change is to the addon resource alone.

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.3.4-0"
```

Comment (gracedo) [reply addons/kommander/1.3/kommander.yaml:12]: I thought the addon revision version was supposed to match the kubeaddons-kommander release version (also `appversion`?) which is `1.3.3`. From what I see in KBA that's how it's done as well, unless we've all been doing it wrong this whole time 😅

Comment (joejulian) [reply addons/kommander/1.3/kommander.yaml:12]: we've been doing it wrong the whole time and we keep getting bit by it.

**Review by joejulian — APPROVED** [review]

File: addons/kommander/1.3/kommander.yaml (hunk 1 above)
Comment (joejulian) [reply addons/kommander/1.3/kommander.yaml:12]: As discussed at standup, I didn't notice that this was a backport. Let's try to get this going on new PRs, not backports.

Comment (gracedo) [reply addons/kommander/1.3/kommander.yaml:12]: ^ FYI @faiq for new autobump PRs across kba/kubeeaddons-kommander (although I suppose these won't be used in 2.x)

Comment (faiq) [reply addons/kommander/1.3/kommander.yaml:12]: Wait so this is supposed to be chart version now?

**Pull request merged**